src/Microsoft.ServiceFabric.Powershell.Http/Generated/ValidateClusterUpgradeCmdlet.cs

```diff
@@ -13,7 +13,7 @@ namespace Microsoft.ServiceFabric.Powershell.Http
     /// <summary>
     /// Validate and assess the impact of a code or configuration version update of a Service Fabric cluster.
     /// </summary>
-    [Cmdlet(VerbsLifecycle.Confirm, "SFClusterUpgrade")]
+    [Cmdlet(VerbsCommon.Validate, "SFClusterUpgrade")]
```

Comment:
@jeffj6123 This code was regenerated with a `Validate` verb which, as far as I can tell, does not exist in any of the Verbs classes defined by PowerShell. This breaks the build, but even if it didn't, this would be a breaking change for PowerShell and need a major new version. Did you encounter this before? Should we revert this?

Reply:
it did this last time this cmdlet got changed, i think its making an assumption on "what" the verb should look like based off swagger so last time I had changed it to confirm after looking over the verbsCommon and validate seemed to fit exactly. I just hadnt changed the code generation yet to reflect this

Reply:
Ok, that's what I thought. Reverted.